app/src/main/res/layout/layout_component_switch.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.SwitchCompat xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/componentSwitch"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginHorizontal="@dimen/base_margin_double"
+    android:layout_marginTop="@dimen/base_margin_double"
+    android:focusable="false"
```

Comment:
How come we're not letting this `Switch` be focusable? 🤔

Reply:
Removed!

Reply:
That is indeed pretty strange 🤔 Looks like `pressImeAction` delegates to https://developer.android.com/reference/androidx/test/espresso/action/ViewActions#pressimeactionbutton.

If this text field is not the last input logically in the form, perhaps the desired platform action is to 'tab' to the next possible input? 🤔

Reply:
I just couldnt get this to work, used 
`onView(ViewMatchers.isRoot()).perform(ViewActions.closeSoftKeyboard())` instead to force close the keyboard in the tests...

Tabbing doesnt feel right since you then focus the switch, which doesnt really need the keyboard.

Reply:
Its just weird to show the keyboard when the switch is focused I think, the "save"-button is then also hidden by the keyboard

Reply:
Agreed.

You can use https://github.com/agoda-com/Kakao/blob/master/docs/kakao/com.agoda.kakao/-screen-actions/close-soft-keyboard.md to dismiss the keyboard instead of using plain espresso though, will make it slightly more readable I think 🙂

Reply:
Cool beans!